concordium-consensus/src/Concordium/ProtocolUpdate/P7/Reboot.hs

```diff
@@ -0,0 +1,111 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeFamilies #-}
+
+-- | This module implements the P7.Reboot protocol update.
+--  This protocol update is valid at protocol version P7, and updates
+--  to protocol version P7.
+--  This produces a new 'RegenesisDataP7' using the 'GDP7Regenesis' constructor,
+--  as follows:
+--
+--  * 'genesisCore':
+--
+--      * 'genesisTime' is the timestamp of the last finalized block of the previous chain.
+--      * 'genesisEpochDuration' is calculated from the previous epoch duration (in slots) times
+--        the slot duration.
+--      * 'genesisSignatureThreshold' is 2/3.
```

Comment:
I think these are just taken from the previous genesis, rather than set as described.

Reply:
I have changed the documentation here